src/styles/header.scss

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+
+  &__logo-link {
+    display: block;
+    height: 40px;
+    width: 40px;
+  }
```

Comment:
For the `.header__logo-link` element, consider using variables for fixed sizes (height and width) if they are meant to be consistent across the project.